macros/models/_sector/transfers/transfers_native.sql

```diff
@@ -0,0 +1,182 @@
+{% macro transfers_native(blockchain, traces, transactions, native_token_address, genesis_balances=null, native_erc_transfers=null) %}
+
+WITH
+
+native_transfers  as (
+        SELECT 
+            'receive' as transfer_type, 
+            tx_hash,
+            trace_address, 
+            block_time,
+            to as wallet_address, 
+            {{native_token_address}} as token_address,
+            CAST(value as double) as amount_raw
+        FROM 
+        {{ traces }}
+        WHERE (call_type NOT IN ('delegatecall', 'callcode', 'staticcall') OR call_type IS NULL)
+        AND success
+        AND CAST(value as double) > 0
+        AND to IS NOT NULL 
+        AND to != 0x0000000000000000000000000000000000000000
+        {% if is_incremental() %}
+            AND block_time >= date_trunc('day', now() - interval '3' Day)
+        {% endif %}
+
+        UNION ALL 
```

Comment:
This is missing ETH transfers from the L1<>L2 Bridge for Optimism Pre-Bedrock (June 6, 2023) (see existing OP ETH Transfers).

We'd need to use the placeholder ERC20 here, and ignore it for erc20 transfers.

Example here: https://optimistic.etherscan.io/tx/0xec2b3a5f852c0595dd167510ede630082ff08fa0bb99ad03ea9de81b3b135f40

Reply:
^ Not sure if any other L2 bridges had similar behavior

Reply:
alternatively: does `native_erc_transfers` handle for this? hard to tell

Reply:
yeah native_erc_transfers handle for that.